openslides/users/models.py

```diff
@@ -158,11 +159,9 @@ def get_absolute_url(self, link='detail'):
         Returns the URL to the user.
         """
         if link == 'detail':
-            url = reverse('user_detail', args=[str(self.pk)])
+            url = "/users/%s/" % self.pk
         elif link == 'update':
-            url = reverse('user_update', args=[str(self.pk)])
-        elif link == 'delete':
-            url = reverse('user_delete', args=[str(self.pk)])
+            url = "/users/%s/edit/" % self.pk
```

Comment:
I don't understand the difference.
If I click on Edit button AngularJS gives me a template. If I click on Delete Button AngularJS does some other magic (confirmation dialog or whethever). Why do we need an URL for the first button but not for the seconde. The Edit template does not have to be connected to an URL. AngularJS can find this without a specific URL if only the button has a respective flag.